pkg/cluster/info.go

```diff
@@ -43,6 +44,7 @@ type Info struct {
 	Name             string
 	RestConfig       *rest.Config
 	ClientProducer   client.Producer
+	Broker           *v1alpha1.Broker
```

Comment:
Structure fields should only be used if their contents have similar validity. When a function receives an `Info` struct, it knows that `Submariner` and `ServiceDiscovery` are fixed — either they’re present, or they’re not, but that’s not going to change, and it doesn’t need to do anything to have the value filled in. `Broker` is different; it’s only valid after `GetBroker` has been called. This means that it’s not a valid structure field; it should be dropped, and `GetBroker()` used whenever the Broker object is needed (with appropriate per-function local storage of results).